projects/samples/devices/controllers/console_capabilities/ansi_codes.h

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
This header file should be useful to other controllers, hence we should include it in the Webots API, and not in a specific controller folder.